modules/apps/foundation/frontend-theme/responsive-boat-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,290 @@
+
+$mobileNavExpandTime: 0.4s;
+$navLinkFadeTime: 0.5s;
+$navBtnPadding: 20px;
+$navBtnHeight: 40px;
+
+/* All styling is nested within #wrapper selector to avoid accidentally styling the liferay menus */
+
+#wrapper {
+	background-attachment: fixed; 		// background stays fixed while scrolling
+	background-color: #A9A9A9; 			// default background when image is not loaded
+	background-image: url("../images/background.jpg");
+	background-position-x: center;
+	background-size: cover;
+	padding: 0; 						// eliminate white spaces on edges of webpage
+	text-align: center; 				// default text-alignment to center all child texts and elements
+
+	#banner {
+		background: rgba(0, 0, 0, 0.5);
+		color: #FFF;
+		padding-bottom: 500px;			// add space below navbar to copy the Wordpress theme
+	}
+
+	/* ---------- NAVIGATION SECTION ---------- */
```

Comment:
CSS comments should be written as sentences with the first word's letter as the only capitalized text.